spec/spec_helper.rb

```diff
@@ -1,4 +1,5 @@
 require 'westfield_datetime_helper'
+require 'pry'
```

Comment:
Will this fail in production? `pry` is only added as a `development_dependency`.